javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+var width = 8;
+var height = 16;
+var area = width * height;
+console.log('value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+var bill = 7;
+var payment = 10;
+var change = payment - bill;
+console.log('value of change: ', change);
+console.log('typeof change: ', typeof change);
+
+var quizzes = 23;
+var midterm = 99;
+var final = 100;
+var grade = (quizzes + midterm + final) / 3;
+console.log('value of grade: ', grade);
+console.log('typeof grade: ', typeof grade);
+
+var firstName = 'Justin';
+var lastName = 'Song';
+var fullName = firstName + ' ' + lastName;
+console.log('value of fullName: ', fullName);
+console.log('typeof fullName: ', typeof fullName);
+
+var pH = 9;
+var isAcidic = pH < 7;
+console.log('value of isAcidic: ', isAcidic);
+console.log('typeof isAcidic: ', typeof isAcidic);
+
+var headCount = 11;
+var isSparta = headCount === 300;
+console.log('value of isSparta: ', isSparta);
+console.log('typeof isSparta: ', typeof isSparta);
+
+var motto = fullName;
+motto = motto + ' is the GOAT';
```

Comment:
Use the addition assignment operator here. `+=`.

Reply:
Updated!